docs/observability/MonitorAzureResources.md

```diff
@@ -0,0 +1,92 @@
+# Azure Resources and Azure Machine Learning Observability
+
+You can collect logs and visualize Azure Resources for observability. In this documentation, we illustrate which option you can use to achieve project goal.
+
+# Observability Architecture
+
+Even though each team may have different access permission to each Azure resource and observability requirements, you can use same technology to collect logs, then customize the visualization and alerts for each team.
+
+![Observability-LanguageModel-20201201](../media/observability-choice.png)
+
+Customer may have their own tools for observabilty. In such case, you can consider log integration between Azure Monitor and their system.
+___
+
+# Azure Machine Learning project group types
+
+Typically, there are at least four types of groups in Machine Learning projects. Each team have different requirements for observability.
+
+- Software Engineer
+- Data Scientist
+- Quality Assuarance
+- Business Analyst
+
+## Software Engineer
+
+SE typically works on entire sysetm and they will monitor application status using [Azure dashboard](https://docs.microsoft.com/en-us/azure/azure-monitor/learn/tutorial-app-dashboards), with alert.
+
+- [Application logs](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/insights-overview) and [system metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-platform#metrics) from all services are exported to [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/overview)
+- Native Azure components export directly to Azure Monitor
+
+## Data Scientist
+
+Data Scientists uses Azure Machine Learning to build and publish models. They should have full access to Azure Machine Learning workspace with detail logs and charts. In addition, they will monitor Model Training and Evaluation results using Azure Dashboards, with alert.
+
+- [Get context in Run environment](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-train-with-datasets#directly-access-datasets-in-your-script) and [add ML train metrics to Application Insight telemetry](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-debug-pipelines#logging-options-example)
+- [Training Pipeline Metrics](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-log-pipelines-application-insights) are exported Application Insights log or OpenCensus format
+
+## Quality Assurance
+
+Quality Assurance team may run the model performance test build by Data Scientist team by using different set of test dataset. You may want to restrice their access to Azure Machine Learning service so that they cannot see the code to build the model, as they just need to run the model accuracy tests. 
```

Comment:
"buil**d** by the Data Scientist..." -> "buil**t** by the Data Scientist..."
"restrice" -> "restict"

Reply:
fixed .